sonar-lits-plugin/src/test/java/com/sonarsource/lits/IssuesCheckerTest.java

```diff
@@ -176,11 +178,13 @@ public void should_fail_when_inactive_rules() {
   }
 
   @Test
-  public void getPrevious_should_return_empty_list_when_no_output_directory() {
+  public void getPrevious_should_return_empty_list_when_no_output_directory() throws IOException {
+    String nonExistingPath = new File("file/does/not/exist").getCanonicalPath();
+
     MapSettings settings = new MapSettings();
-    settings.setProperty(LITSPlugin.OLD_DUMP_PROPERTY, "/file/does/not/exist");
-    settings.setProperty(LITSPlugin.NEW_DUMP_PROPERTY, "/file/does/not/exist");
-    settings.setProperty(LITSPlugin.DIFFERENCES_PROPERTY, "/file/does/not/exist");
+    settings.setProperty(LITSPlugin.OLD_DUMP_PROPERTY, nonExistingPath);
```

Comment:
Note: I updated this test as it errored out on Windows, as `file/does/not/exist` was not a valid absolute path there.